clients/wallet/src/horizon/tests.rs

```diff
@@ -89,7 +91,7 @@ async fn build_simple_transaction(
 async fn horizon_submit_transaction_success() {
 	let horizon_client = reqwest::Client::new();
 
-	let source = secret_key_from_encoding(SECRET);
+	let source = secret_key_from_encoding(&get_source_secret_key_from_env(false));
```

Comment:
While not necessary, it might be a good idea to define a new variable`let is_public_network = false` at the beginning of each test function and then replacing it also in all the function calls that currently directly use a `false` boolean. This way, we make it more explicit and make it more robust in case we decide to change the used network in the future.